testsuite/MDAnalysisTests/utility/pylibxdr.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+import numpy as np
+
+
+def unpack_array(xdr, shape):
+    arr = np.array([xdr.unpack_float() for _ in range(np.prod(shape))])
+    return arr.reshape(shape)
+
+
+# Poor mans namespace
+class XTC:
```

Comment:
This is a partial python implementation of the xtc and trr file formats. It is useful to generate corrupt xtc files. to trigger specific warnings